File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_portdelrn.py
"""Reference emit for PORTDELRN — PORTDEL reason EVALUATE TRUE → 60."""
# EXPECTED: 60
REASONS = ["01", "02", "03"]
CODES = {"01": 10, "02": 20, "03": 30}
total = sum(CODES[r] for r in REASONS)
print(f"{total:02d}")

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_portdelrn.cs
// EXPECTED: 60
// Reference emit for PORTDELRN — PORTDEL reason EVALUATE TRUE RC sum.
using System;
using System.Collections.Generic;
public static class PortdelrnRef {
  public static void Main() {
    var reasons = new[] { "01", "02", "03" };
    var codes = new Dictionary<string, int> { ["01"] = 10, ["02"] = 20, ["03"] = 30 };
    int total = 0;
    foreach (var r in reasons) total += codes[r];
    Console.WriteLine($"{total:D2}");
  }
}
